build-image/first-boot.sh

```diff
@@ -33,7 +33,11 @@ else
   echo "OK"
 fi
 
-userdef="pi"
+userdef="openhabian"
+if is_pi; then
```

Comment:
@EliasGabrielsson did I overlook the function definition?

Reply:
Sorry but I don't understand what you mean?

Reply:
Where is `is_pi` defined? Sorry if I overlooked it.

Reply:
No worries, it's in the commit message:

"Adds dependency to `functions/helpers.sh`"

referred you on the specific places